package.json

```diff
@@ -350,6 +350,12 @@
                 "command": "azureFunctions.viewProperties",
                 "title": "%azureFunctions.viewProperties%",
                 "category": "Azure Functions"
+            },
+            {
+                "command": "azureFunctions.eventGrid.sendMockRequest",
```

Comment:
Is there anyway we can define the order of the buttons? 
![image](https://github.com/microsoft/vscode-azurefunctions/assets/5290572/588f597b-8206-4c9d-b841-1a7ca21d8280)

Because it's in the middle, I find it a little less obvious a new button was contributed to this ribbon.

Reply:
I second this. I believe there should be a way to move it to the left.

Reply:
Discussed offline with @nturinski. I agree and I tried to make it as far left as possible, I'm not aware of any ways to ensure that's the case. I think it may be related to which extensions load first, because in my experience, the "save and send request" button appears a few seconds before the git diff button. Do you know of a way to ensure it's always on the farthest left?

Reply:
I was messing with it for a bit too and didn't see anyway to guarantee it. I was looking at [this](https://code.visualstudio.com/api/references/contribution-points#Sorting-of-groups) and it didn't really seem like it had anything useful.

Reply:
Resolving for now unless someone can suggest a way to ensure it is also at the farthest left